src/wallet/wallet.cpp

```diff
@@ -3659,15 +3659,11 @@ void CWallet::MarkReserveKeysAsUsed(int64_t keypool_id)
             m_pool_key_to_index.erase(keypool.vchPubKey.GetID());
         }
         walletdb.ErasePool(index);
+        LogPrintf("keypool index %d removed\n", index);
```

Comment:
This matches the existing 'keypool keep' logs, which are not wallet debug category.

This PR already has a bunch of ACKs and it's wanted for rc2, so I'd rather not change this now. I'll happily open a PR to make this (and other wallet logs) wallet debug category.